lib/vagrant-vmck/action/create.rb

```diff
@@ -14,15 +14,20 @@ def initialize(app, env)
         def call(env)
           client = env[:vmck]
 
-          env[:ui].info("Vmck starting job ...")
-          options = {
-            'cpus': env[:machine].provider_config.cpus,
-            'memory': env[:machine].provider_config.memory,
-            'image_path': env[:machine].provider_config.image_path,
-            'storage': env[:machine].provider_config.storage,
-            'name': env[:machine].provider_config.name,
-          }
-          id = client.create(options)['id'].to_s
+          if ENV['VMCK_JOB_ID'].nil?
+            env[:ui].info("Vmck starting job ...")
+            options = {
+              'cpus': env[:machine].provider_config.cpus,
+              'memory': env[:machine].provider_config.memory,
+              'image_path': env[:machine].provider_config.image_path,
+              'storage': env[:machine].provider_config.storage,
+              'name': env[:machine].provider_config.name,
+            }
+            id = client.create(options)['id'].to_s
+          else
+            id = ENV['VMCK_JOB_ID'].to_s
```

Comment:
Add a logging message saying "Vmck using existing job #123", because the other `if` branch has one too. And you don't need `.to_s` because it's already a string.